src/Domain/LeanCode.CQRS.Security/CQRSSecurityElement.cs

```diff
@@ -29,6 +29,8 @@ public CQRSSecurityElement(IAuthorizerResolver<TAppContext> authorizerResolver)
 
             if (customAuthorizers.Count > 0 && !(user?.Identity?.IsAuthenticated ?? false))
             {
+                logger.Warning("Unauthenticated user requested object that requires authorization");
```

Comment:
The exception thrown just below will result in an error logged just after this anyway.

Reply:
Not a case anymore.